uint/tests/uint_tests.rs

```diff
@@ -56,6 +56,13 @@ fn uint256_checked_ops() {
 	let a = U256::from(10);
 	let b = !U256::from(1);
 
+	assert_eq!(U256::from(10).checked_pow(U256::from(0)), Some(U256::from(1)));
+	assert_eq!(U256::from(10).checked_pow(U256::from(1)), Some(U256::from(10)));
+	assert_eq!(U256::from(10).checked_pow(U256::from(2)), Some(U256::from(100)));
+	assert_eq!(U256::from(10).checked_pow(U256::from(3)), Some(U256::from(1000)));
+	assert_eq!(U256::from(10).checked_pow(U256::from(20)), Some(U256::exp10(20)));
```

Comment:
```suggestion
	assert_eq!(U256::max_value().checked_pow(U256::from(2)), None);
```
I believe this is what David suggested to test.

Reply:
Added below. Let me know if you have further suggestions.